e2e/tests/ui-driven/src/invite-to-pay/mocks.ts

```diff
@@ -65,6 +65,7 @@ export const mockSessionData: Omit<SessionData, "id"> = {
       "proposal.projectType": ["alter.decks", "alter.internal"],
       "applicant.agent.email": "testAgent@opensystemslab.com",
       "application.fee.payable": 123.45,
+      "application.fee.calculated": 123.45,
```

Comment:
Running regression tests against this branch, suspect this should fix our error from last night! Payable no longer automatically sets calculated, both should exist in passport if breakdown includes both

https://github.com/theopensystemslab/planx-new/actions/runs/14905059788